bin/load-test-db

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# no more tables
+PGPASSWORD=something_secret psql -U postgres -h localhost -p 5432 -d ttasmarthub -c "DROP SCHEMA public CASCADE; CREATE SCHEMA public;"
```

Comment:
makes me think I should have included this earlier
todo - maybe ```source .env``` or however you do it to load the particular creds people might have set up